src/Horus/Global.hs

```diff
@@ -347,8 +352,22 @@ solveContract = do
     )
       infos
  where
-  isStandardSource :: Set ScopedFunction -> ScopedFunction -> Bool
-  isStandardSource inlinables f = f `notElem` inlinables && not (isWrapper f)
+  standardSources :: Set ScopedFunction -> [LabeledInst] -> GlobalL (Set ScopedFunction)
+  standardSources inlinableFs rows =
+    getProgram >>= \prog ->
+      let functionsWithBodies = functionsOf rows prog
+          functions = Map.keys functionsWithBodies
+          (cg, vToLbl, lblToV) = graphOfCG . callgraph . Map.mapKeys sf_pc $ functionsWithBodies
+          (wrapperFunctions, nonwrapperFunctions) = partition isWrapper functions
+          reachableLsFromWrappers =
```

Comment:
Silly, but can we spell-out `Labels` in and around here? I'm not sure the abbreviation gets us much since `fourmolu` doesn't break lines anyway.